src/main/java/HotJavaController.java

```diff
@@ -8,13 +8,11 @@
 
 
 /**
- * The controller for HotJava REST endpoints and web UI
- * <p>
- * This class handles the CRUD operations for HotJava Vehicles, via HTTP actions.
- * </p>
- * <p>
- * This class also serves HTML based web pages, for UI interactions with vehicles.
- * </p>
+ * TODO
+ * Create a controller for HotJava REST endpoints and web UI
+ * Create a class handles the CRUD operations for HotJava Vehicles, via HTTP actions.
+ * which class also serves HTML based web pages, for UI interactions with vehicles.
+ *
```

Comment:
Code Review Input # 1 - Comment changes
This comment should be cleaned up, as it is very messy and inconsistent with other comments in the application.

To give it more meaning, I removed the paragraph tags as well as changing the text itself to allow more context and guidance for when a developer goes to work on that page later.

This block of code is an Implementation comment in the form of a block. It is being used to describe code that needs to be implemented in the future, which is why I included "TODO", to also keep consistency with other areas where the developers did this.

References: 
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3
https://www.baeldung.com/java-clean-code

Reply:
I'd leave this as it were.   HTML tags are permitted in JavaDoc, and can help to format the comments.